.github/ISSUE_TEMPLATE/20-documentation-issue.md

```diff
@@ -4,14 +4,34 @@ about: Use this template for documentation related issues
 
 ---
 
-<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>
+<em>Thanks so much for taking the time to file a documentation issue and even more thanks if you intend to contribute to updating it! Please do introduce yourself on our Mailing List via [Google Groups](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs) or [email](docs@tensorflow.org list) and let us know if you have any questions. We also encourage you to review our [Documentation Contributor Guide](https://www.tensorflow.org/community/contribute/docs). As a side note, per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>
 
+## Existing URLs containing the issue:
+Link to the documentation entry, for example: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/MyMethod
 
-**System information**
-- TensorFlow version:
-- Doc Link:
+## Description of issue (what needs changing):
 
+## Correct Links
+Is the link pointing to the source code correct? 
+Hint, try to `git grep my_method` here: (https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/)
 
-**Describe the documentation issue**
+## Clear Description
+Why should someone use this method? How is it useful?
 
-**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
+## Usage Example
+Is there a usage example?
+
+## Parameters Defined
+Are all arguments that can be passed in defined and formatted correctly?
+
+## Returns Defined
+Are return values defined?
+
+## Raises Listed and Defined
+Are errors defined? link to raises
+
+## Visuals, if Applicable
```

Comment:
We don't have a clear way to contribute diagrams or images to tensorflow.org. Usually, we try to keep them out of git since it bloats the repo (but some subsites do host images within their repo).
So perhaps clarify this field as "Request visuals, if applicable"